portfolio/src/main/webapp/scripts/comments.js

```diff
@@ -1,3 +1,12 @@
+/**
+ * Disable a submit button when the form is submitted.
+ */
+function disableSubmitButton(event) {
+  let button = event.currentTarget.querySelector(".submit");
```

Comment:
Nit: why not search for submit-comment instead?

Reply:
Because this way the function can be used to disable any form that has a submit button with class submit.

Reply:
ah, very nice, thanks.